app/models/generic_object.rb

```diff
@@ -35,14 +35,39 @@ def property_attributes
     end
   end
 
+  def delete_property(name)
+    properties.delete(name.to_s)
+    save
+  end
+
+  def add_to_property_association(name, objs)
+    objs = [objs] unless objs.kind_of?(Array)
+    name = name.to_s
+    properties[name] ||= []
+
+    selected = objs.select { |obj| obj.kind_of?(property_associations[name].constantize) }
+    properties[name] = (properties[name] + selected.pluck(:id)).uniq if selected
+    save if properties_changed?
```

Comment:
I got it. But I don't think we need to check the `changed?` flag every time when we attempt to save some field.